.travis.yml

```diff
@@ -42,6 +43,19 @@ before_install:
     # Set the BOOST_ROOT environment variable.
     - export BOOST_ROOT=$HOME/boost_1_61_0
     - export LD_LIBRARY_PATH=$HOME/boost_1_61_0/stage/lib:$LD_LIBRARY_PATH
+    
+    # Download, build and install Awesomium and its dependencies
+    - wget -O libgcrypt.deb http://security.ubuntu.com/ubuntu/pool/main/libg/libgcrypt11/libgcrypt11_1.5.0-3ubuntu0.6_amd64.deb
+    - sudo dpkg -i libgcrypt.deb    
+    
+    - wget -O libudev.deb http://mirrors.kernel.org/ubuntu/pool/main/u/udev/libudev0_175-0ubuntu9_amd64.deb
+    - sudo dpkg -i libudev.deb
+    
+    - wget -O awesomium.tar.gz http://www.awesomium.com/downloads/awesomium_1_7_5_sdk_linux64.tar.gz
+    - tar -xvzf awesomium.tar.gz 
+    - cd awesomium_v1.7.5_sdk_linux64/
```

Comment:
Oh, I know this one! You can get rid of the `cd`s and replace the `make` command by `sudo make -C awesomium_v1.7.5_SDK_linux64 install`. That'll save you two lines. :)